### ISSUE TITLE
New repo: sagemath/mols-handbook-data.git

### ISSUE THREAD

**Issue by orlitzky** [sagemath/sage#37025]
### Problem Description

As part of https://github.com/sagemath/sage/issues/30914, I've created https://github.com/orlitzky/mols-handbook-data to replace the old _combinatorial_designs_ package with something that can be detected by python.

### Proposed Solution

Can I please have access to a new `sagemath/mols-handbook-data.git` repo to push my changes? This is required before the first release if we want the docs to point to the right place.

### Alternatives Considered

N/A

### Is there an existing issue for this?

- [X] I have searched the existing issues for a bug report that matches the one I want to file, without success.

**Comment by tornaria** [comment]
Please don't. This is just a single text file which contains a bunch of number copied from a handbook. It's not even *used* in sagemath, except for testing purposes.

See what I did in #37024.

**Comment by tornaria** [comment]
Just don't, please. There's really no need to fragment everything in separate packages just because. It's not like the file is going anywhere, even if sagemath were to drop it.

I think it's a complete waste of time (yours and others, now and in the future).

Why not have a `sage-is-odd` package? Node has it, and it's quite successful (https://www.npmjs.com/package/is-odd -- 200k downloads *per week*)

**Comment by orlitzky** [comment]
> Just don't, please. There's really no need to fragment everything in separate packages just because. It's not like the file is going anywhere, even if sagemath were to drop it.
> 
> I think it's a complete waste of time (yours and others, now and in the future).

It's already a separate package, a nonstandard one that is coupled to sage but can't be discovered from within sage. The last time anyone touched this stuff was a decade ago, and the work to package it is already done. So whatever time was going to be wasted has already been wasted.

And in return, we get

* Simpler distro packages (compared to the existing database packages that are married to sage)
* Reusability outside of sage
* The ability to easily drop it as a dependency of sage in the future
* (Relative to #37024) a file filled with 10,000 integers that is touched once every ten years stays out of our source tree

> Why not have a `sage-is-odd` package? Node has it, and it's quite successful (https://www.npmjs.com/package/is-odd -- 200k downloads _per week_)

Sage is already odd enough. The new package isn't tied to sage any more and would be simply `is-odd`, which, apparently, is quite popular. If 200,000 people wanted to look up lower bounds for mutually-orthogonal latin squares in python, that would be a great reason to make it a separate package.

**Comment by tornaria** [comment]
> > Just don't, please. There's really no need to fragment everything in separate packages just because. It's not like the file is going anywhere, even if sagemath were to drop it.
> > I think it's a complete waste of time (yours and others, now and in the future).
> 
> It's already a separate package, a nonstandard one that is coupled to sage but can't be discovered from within sage. The last time anyone touched this stuff was a decade ago, and the work to package it is already done. So whatever time was going to be wasted has already been wasted.

You are certainly entitled to waste your own time as you wish. But making a package for this will waste other people time for (IMO) no gain. Distros will have to repackage a stupid package (current combinatorial_designs) into another stupid package, with all the deprecation, replacement, etc. It's another package for which sagemath becomes upstream, another package in pypi, etc.

>     * Simpler distro packages (compared to the existing database packages that are married to sage)

Distro package is already there. It's just files placed somewhere. With #37024 the files can be found either in `/usr/share/<DATABASE NAME>` or in `/usr/share/sagemath/<DATABASE NAME>`, or even in `/usr/local/share/sagemath/<DATABASE NAME>` or even in `$HOME/.sage/db/<DATABASE NAME>` installed by the user if missing from the system.

This for a "real" database (e.g. cremona's database of elliptic curves which is quite big). In this case, distro package can be removed after #37024.

Nothing is simpler than nothing.

Note that the `build/pkgs/combinatorial` directory uses more storage blocks than the actual file (!!). Also, there are more tables/databases in sage source (`src/sage/combinat/designs/database.py`, `src/sage/knots/knot_table.py` just to name two random ones). Are you going to make a python package for each one of them? Are you going to force sagemath into being upstream for all of that?

>     * Reusability outside of sage

Not useful for anything. ¿Do you use it? ¿Who will use it? ¿Who will maintain it? It's just a list of numbers, let's keep it a list of numbers in a place where it's used (and tested, fwiw: how are you going to test it, sure your package will not `checkdepends` on sagemath).

>     * The ability to easily drop it as a dependency of sage in the future

It becomes a file after #37024 and one can just `/bin/rm` it (or `git rm` if you will).

>     * (Relative to [Access database and other files through features, for simpler configuration #37024](https://github.com/sagemath/sage/pull/37024)) a file filled with 10,000 integers that is touched once every ten years stays out of our source tree

It stays in the only place it has a chance to be useful, if any.


> > Why not have a `sage-is-odd` package? Node has it, and it's quite successful (https://www.npmjs.com/package/is-odd -- 200k downloads _per week_)
> 
> Sage is already odd enough. The new package isn't tied to sage any more and would be simply `is-odd`, which, apparently, is quite popular. If 200,000 people wanted to look up lower bounds for mutually-orthogonal latin squares in python, that would be a great reason to make it a separate package.

:shrug:

**Comment by orlitzky** [comment]
I've never gotten so much nothing done in my life, talk about wasted time.
